apps/cli/lib/block_provider/rpc.ex

```diff
@@ -148,6 +148,9 @@ defmodule CLI.BlockProvider.RPC do
   @spec load_new_block(integer(), ethereumex_client(), integer()) :: {:ok, %{}} | {:error, any()}
   defp load_new_block(number, client, retries \\ @max_retries) do
     case client.eth_get_block_by_number(to_hex(number), true) do
+      {:ok, nil} ->
```

Comment:
Can we change the response to not be `{:ok, nil}`? If `:ok`, I'd expect a block since the request is for a block.

Reply:
@willmeister it's parity and geth who return `{:ok, nil}`